blivet/formats/luks.py

```diff
@@ -250,7 +256,8 @@ def _create(self, **kwargs):
                                     key_file=self._key_file,
                                     cipher=self.cipher,
                                     key_size=self.key_size,
-                                    min_entropy=self.min_luks_entropy)
+                                    min_entropy=self.min_luks_entropy,
+                                    luks_version=crypto.LUKS_VERSIONS[self.luks_version])
```

Comment:
Ideally, `luks_version` would be a property with the check from `__init__` moved into the setter.